docs/tools/vdb_table/data/oramasearch.json

```diff
@@ -25,15 +25,15 @@
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 6700,
+  "github_stars": 7100,
   "vector_launch_year": 2022,
   "metadata_filter": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "none",
+    "support": "full",
```

Comment:
please provide a reference to docs/example notebook for this in "source_url". It should show ability to merge between ranked lists of keyword search and vector search results.